plugins/modules/ipa_dnsrecord.py

```diff
@@ -35,13 +35,14 @@
   record_type:
     description:
     - The type of DNS record name.
-    - Currently, 'A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'NS', 'PTR', 'TXT', 'SRV' and 'MX' are supported.
+    - Currently, 'A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'NS', 'PTR', 'TXT', 'SRV', 'MX' and 'SSHFP' are supported.
     - "'A6', 'CNAME', 'DNAME' and 'TXT' are added in version 2.5."
     - "'SRV' and 'MX' are added in version 2.8."
     - "'NS' are added in comunity.general 8.2.0."
+    - "'SSHFP' are added in community.general 9.0.1"
```

Comment:
No new features in bugfix releases :)
```suggestion
    - "'SSHFP' are added in community.general 9.1.0"
```
(The whole description needs some improvements, but it's better to do that in a separate PR.)